buildspec.yml

```diff
@@ -14,7 +14,7 @@ phases:
       - 'echo "Run jscs..."'
       - './node_modules/.bin/gulp jscs'
       - 'echo "Package as ZIP file"'
-      - 'zip -r "suitec-preview-service.zip" * -x logs\*'
+      - 'zip -r "suitec-preview-service.zip" * .e* .r*'
```

Comment:
@johncrossman Also, is the goal to zip up exactly what's under version control and nothing else; or are we including npm-installed dependencies and such in the artifact? If the former, `git archive` might be a good tool: https://coderwall.com/p/fwvttg/export-a-git-branch-to-zip